presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkTaskExecutorFactory.java

```diff
@@ -561,6 +561,9 @@ public boolean hasNext()
         private Tuple2<MutablePartitionId, T> doComputeNext()
                 throws InterruptedException
         {
+            if (start == null) {
+                start = System.currentTimeMillis();
```

Comment:
So before we actually under-estimate our throughput right? ;)

Reply:
Yeah. A little bit. It wasn't very significant, but I decided it is better to fix it.